src/nni_manager/training_service/local/gpuScheduler.ts

```diff
@@ -85,9 +85,13 @@ class GPUScheduler {
 
     public async stop() {
         this.stopping = true;
-        const pid: string = await fs.promises.readFile(path.join(this.gpuMetricCollectorScriptFolder, 'pid'), 'utf8');
-        await cpp.exec(`pkill -P ${pid}`);
-        await cpp.exec(`rm -rf ${this.gpuMetricCollectorScriptFolder}`);
+        try {
+            const pid: string = await fs.promises.readFile(path.join(this.gpuMetricCollectorScriptFolder, 'pid'), 'utf8');
```

Comment:
Suggest to consider how to show errors in NNI cleanup to end-user, but you can improve later after v0.6 release.

Reply:
ok, will refactor the logic in next release.